includes/Core/Modules/Modules.php

```diff
@@ -428,6 +430,26 @@ function( Module $module ) {
 		return $data;
 	}
 
+	/**
+	 * Populates modules data to pass to JS.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $modules_data Inline modules data.
+	 * @return array Inline modules data.
+	 */
+	private function inline_modules_data( $modules_data ) {
+		$available_modules = $this->get_available_modules();
+
+		foreach ( $available_modules as $module ) {
+			if ( $module instanceof Module_With_Data_Available_State ) {
+				$modules_data[ 'data_available_' . $module->slug ] = $this->is_module_active( $module->slug ) && $module->is_data_available();
```

Comment:
I a previous suggestion here may have been lost, but we also want to check that the module is connected here, so that the value is `active + connected + data_available`.

Reply:
Sorry for the misunderstanding. Fixed!